script/cpplint.js

```diff
@@ -8,12 +8,30 @@ const minimist = require('minimist')
 const path = require('path')
 
 const SOURCE_ROOT = path.normalize(path.dirname(__dirname))
-const LINTER_PATH = path.join(SOURCE_ROOT, 'vendor', 'depot_tools', 'cpplint.py')
 
-function callCpplint (filenames, args) {
-  if (args.verbose) console.log([LINTER_PATH, ...filenames].join(' '))
+function findCppLint () {
+  const linterName = 'cpplint.py'
+
+  const isGNBuild = fs.existsSync(SOURCE_ROOT, '..', '..', '.gclient')
+  if (isGNBuild) return linterName // GN users have depot_tools in $PATH
+
+  const gypPath = path.join(SOURCE_ROOT, 'vendor', 'depot_tools', linterName)
```

Comment:
Why would you need this in the master?

Reply:
Hm good point.

My initial thinking was that it would be useful to have a script that worked regardless of which branch it was in, e.g. if we backport it. But I didn't think it through; this doesn't rise to the level of severity needed to backport